checkbox-ng/plainbox/impl/ctrl.py

```diff
@@ -236,8 +288,63 @@ def get_inhibitor_list(self, session_state, job):
                     related_job=dep_job_state.job,
                 )
                 inhibitors.append(inhibitor)
+        if job.id in [Suspend.AUTO_JOB_ID, Suspend.MANUAL_JOB_ID]:
+            for inhibitor in self._get_suspend_inhibitor_list(
+                session_state, job
+            ):
+                inhibitors.append(inhibitor)
         return inhibitors
 
+    def _get_suspend_inhibitor_list(self, session_state, suspend_job):
```

Comment:
The code in this function looks very similar to the one above it. Same things apply + outlining of predicates will help emphasize the differences between functions.

Reply:
I meant similar to `_get_before_suspend_dependency_set`